### PR TITLE
fix isAdmin for group toggles

### PR DESCRIPTION
**Pull request by peter-horvath**

### CODE REVIEW

**Comment by craigjbass** [comment]
This fix can go live once we have segments working for group toggles

File: app/scripts/config/config.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 var config = {
-    jwtServer: '{{ jwt_server_url }}',
-    accountApiHost: '{{ account_api_host }}',
-    accountApiEndpoint: '{{ account_api_endpoint }}'
+    jwtServer: 'https://secure.clearbooks.local/accounting/auth/auth?app=labs',
```
Comment (MeliaBurnley) [inline app/scripts/config/config.js]: you're gonna want to revertaroo this

Comment (peter-horvath) [inline app/scripts/config/config.js]: Oh dear... yes indeed, sorry.

**Comment by craigjbass** [comment]
I manually tested this and it seems to work :+1:

**Pull request merged**